pages/common/fd.md

```diff
@@ -1,6 +1,6 @@
 # fd
 
-> A simple, fast and user-friendly alternative to `find`.
+> An alternative to `find`.
```

Comment:
I guess removing them is a good thing to make the pages more impartial, but this now leaves me asking "why would I use it? What's different about it compared to `find`?"

Reply:
I see your concern and agree with all of your comments. It's a matter of rephrasing the descriptions properly. For this specific case, though, I have no idea of how to phrase it differently.

Reply:
Hrm. Taking a look at the README, what about this?
```suggestion
> An alternative to `find`.
> Aims to be faster and easier to use than `find`.
```

Reply:
Looks good.